Resources/Prototypes/_NF/Shipyard/mayflower.yml

```diff
@@ -0,0 +1,28 @@
+- type: vessel
+  id: Mayflower
+  name: USS Mayflower
+  description: An oldership from an Older world found cover in Space Spiders and Web
+  price: 50000
+  category: Medium
+  group: Expedition
+  shuttlePath: /Maps/Shuttles/Mushroom.yml
+  
+- type: gameMap
+  id: Mayflower
+  mapName: 'USS Mayflower'
+  mapPath: /Maps/Shuttles/Mushroom.yml
+  minPlayers: 0
+  stations: 
+    Mayflower:
+      stationProto: StandardFrontierExpeditionVessel
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: 'USS Mayflower{1}'
```

Comment:
Name format missing space

Reply:
what do you mean by that? Where is the space post to be at?

Reply:
> what do you mean by that? Where is the space post to be at?

between Mayflower and {1}

Reply:
ooo ok also... i just want to say I appearnlty mess this up really big so I am right now fixing all of this with the help of minty